src/domain/space/space.defaults/space.defaults.service.ts

```
@@ -1,156 +1,133 @@
+            await this.templatesManagerService.getTemplateFromTemplateDefault(
+              platformTemplatesManager.id,
+              TemplateDefaultType.PLATFORM_SPACE
+            );
+          templateID = levelZeroTemplate.id;
+          break;
+        case SpaceType.KNOWLEDGE:
+          const knowledgeTemplate =
+            await this.templatesManagerService.getTemplateFromTemplateDefault(
+              platformTemplatesManager.id,
+              TemplateDefaultType.PLATFORM_SUBSPACE_KNOWLEDGE
+            );
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)